wbia/control/_sql_helpers.py

```diff
@@ -376,7 +379,8 @@ def _check_superkeys():
             ), 'ERROR UPDATING DATABASE, SUPERKEYS of %s DROPPED!' % (tablename,)
 
     logger.info('[_SQL] update_schema_version')
-    db_fpath = db.fpath
+    # Remove file:// from db.uri
+    db_fpath = db.uri[len('file://') :]
```

Comment:
I'd prefer to change this to a `.lstrip('file://')` or a `replace('file://', '')`.  I'd also love to do an assert or an exists check (`if 'file://' in db.uri`, etc) before doing a string indexing like this

Reply:
`lstrip` does not work the way one would expect:

```
>>> 'file:///home/karen/'.lstrip('file://')
'home/karen/'
```

but `replace` is fine:

```
>>> 'file:///home/karen/'.replace('file://', '')
'/home/karen/'
```

If we're using `replace` then there's no need to do `if 'file://' in db.uri`.

I'll add a fixup commit to use `replace`